lib/kast.js

```diff
@@ -322,12 +372,12 @@ const format = (o, isRaw = false, mixFix = false, colorize = false) => {
       , "_^Word__EVM-DATA"   : " ^W "
       , "_+Map__RULES"       : " +M "
       , "_andBool_"          : " AND "
-      , "_orBool__BOOL"      : " OR " 
-      , "_:__EVM-DATA"       : " : "  
+      , "_orBool__BOOL"      : " OR "
+      , "_:__EVM-DATA"       : " : "
       , "_==K_"              : " ==K "
-      , "#And"               : "\n"   
+      , "#And"               : "\n"
```

Comment:
When is `#And` used?

Reply:
not sure, it was introduced long long time ago, i think k used to return different kinds of and since and was treaded weirdly by k, maybe they got over it and we can experiment with removing it, but plz in a separate issue

it wa sintroduced here: https://github.com/dapphub/klab/commit/f5b6ea6e218d9e11b95ab42da5f881dbfa0c79b8

Reply:
not sure, it was introduced long long time ago, i think k used to return different kinds of and since and was treaded weirdly by k, maybe they got over it and we can experiment with removing it, but plz in a separate issue

it wa sintroduced here: https://github.com/dapphub/klab/commit/f5b6ea6e218d9e11b95ab42da5f881dbfa0c79b8